### DISCUSSION TITLE
Does this package work with the thop package?

### DISCUSSION THREAD

**Discussion by vedantroy** [arogozhnikov/einops#200] [General]
Hi! I was wondering if this package would work with the thop (https://github.com/Lyken17/pytorch-OpCounter) package, which counts the flops in one's model.

**Comment by arogozhnikov** [comment]
Most likely yes. Better try and check. If thop properly counts things for simple layers, it would work correctly with einops too. Einops completely relies on torch operations for tensor manipulations.

Important remark: most einops operations, like rearrange do no contribute to flops count and have no parameters. 
These are data movement and those are not counted as FLOPS because no arithmetic operations involved
